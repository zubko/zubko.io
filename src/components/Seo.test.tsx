import { StaticQuery } from "gatsby";
import renderer from "react-test-renderer";
import { Seo } from "./Seo";

beforeEach(() => {
  StaticQuery.mockImplementationOnce(({ render }) =>
    render({
      site: {
        siteMetadata: {
          title: "site title",
          description: "site description",
          author: "author"
        }
      }
    })
  );
});

describe("Seo", () => {
  it("most common usage renders without regressions", () => {
    const tree = renderer
      .create(<Seo title="Title" keywords={["keyword-1", "keyword-2"]} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
