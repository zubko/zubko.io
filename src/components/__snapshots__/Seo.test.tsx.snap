// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seo most common usage renders without regressions 1`] = `null`;
